lampod-cli/src/main.rs

```diff
@@ -158,8 +158,17 @@ async fn run(args: LampoCliArgs) -> error::Result<()> {
         }
     };
 
+    let wallet = Arc::new(wallet);
+
     log::debug!(target: "lampod-cli", "wallet created with success");
-    let mut lampod = LampoDaemon::new(lampo_conf.clone(), Arc::new(wallet));
+    let mut lampod = LampoDaemon::new(lampo_conf.clone(), wallet.clone());
+
+    tokio::spawn(async move {
+        // FIXME: the sync should have a tick that redo the check every x seconds
```

Comment:
probably worth doing it now, probably `wallet.listen()` and then into it make the logic for the sync